src/Files.App/Utils/Storage/Enumerators/UniversalStorageEnumerator.cs

```diff
@@ -254,7 +254,7 @@ private static async Task<IReadOnlyList<IStorageItem>> EnumerateFileByFile(BaseS
 			var itemSizeBytes = basicProperties.Size;
 			var itemType = file.DisplayType;
 			var itemFileExtension = file.FileType;
-			var itemThumbnailImgVis = false;
```

Comment:
Not something I'm a fan of. Another person looking at this, wanting to use the variable might think the changes was done intentionally

Reply:
@gumbarros

Reply:
The constant helped to see the bigger picture, when I was about to remove, I seen this:
![image](https://github.com/files-community/Files/assets/52143624/6daa511b-4ab7-407a-b196-5c0c274f2bfd)
`LoadFileIcon` should just be false if the constant is hardcoded right?